www-client/luakit/luakit-2.4.0.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-1 luajit )
+
+inherit lua-single toolchain-funcs xdg
+
+DESCRIPTION="A fast, extensible, and customizable web browser"
+HOMEPAGE="https://luakit.github.io/luakit"
+
+SRC_URI="https://github.com/luakit/luakit/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3+"
+
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+IUSE="doc test"
+
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
```

Comment:
I see an extra dependency (`lua-socket`) when diffing the old/new versions?

Reply:
Indeed, but it was not mandatory (gopher support), but I added it anyway.